### PR TITLE
ZEN-32510 upgrade twisted lib to 19.10.0

### PR DESCRIPTION
**Pull request by romanbakaleyko**

### CODE REVIEW

**Comment by webhookbotuser** [comment]
Can one of the admins verify this patch?

**Comment by romanbakaleyko** [comment]
added https://pypi.org/project/bcrypt/ (License: Apache Software License (Apache License, Version 2.0)) as despite twisted doc says it is optional I get import errors when build zenoss appliaction.

File: requirements_3rd.txt [hunk 1]
```diff
@@ -6,9 +6,10 @@ amqplib==1.0.2
 anyjson==0.3.3
 astrolabe==0.4.0
 atomic==0.7.2
-attrs==16.0.0
+attrs==17.4.0
 Babel==2.3.4
 backports.ssl-match-hostname==3.4.0.1
+bcrypt==3.1.7
```
Comment (jpeacock-zenoss) [inline requirements_3rd.txt]: Why is "bcrypt" being added to the list?
Twisted does not have a dependency on this package.

**Comment by romanbakaleyko** [comment]
retest this

**Pull request merged**